### ISSUE TITLE
Stuck audio connections. AudioManager#closeAudioConnection does nothing.

### ISSUE THREAD

**Issue by Kodehawa** [DV8FromTheWorld/JDA#456]
There's a bug in JDA where calling AudioManager#closeAudioConnection will do nothing, the bot reports it's not connected to *any* channel and it refuses to play anything until: 

- a) You move the bot to a different voice channel and completely delete the channel.
- b) You restart the shard
- c) You run the following code: 

```java
JDAImpl api = (JDAImpl) event.getJDA();
api.getClient().send(new JSONObject().put("op", WebSocketCode.VOICE_STATE).put("d", new JSONObject()
    .put("guild_id", guildId)
    .put("channel_id", JSONObject.NULL)
    .put("self_mute", false)
    .put("self_deaf", false)).toString());
```

**Comment by MinnDevelopment** [comment]
I am assuming this will be fixed in #457 and request testing.

**Comment by MinnDevelopment** [comment]
This should be fixed as of build `270`. If still occurring please re-open this issue.
I advise not to send the mentioned payload on the new builds as it will make connections seem to come out of sync to users.
